omod/src/main/java/org/openmrs/module/metrics/util/MetricHandler.java

```diff
@@ -38,32 +41,38 @@ public void buildMetricFlow(LocalDateTime startRange, LocalDateTime endRange) th
 		Map<String, Integer> noOfEncounters = metricService.getEncounterObjectTypesCountByGivenDateRange(startRange,
 		    endRange);
 		
+		ObjectName objectName;
+		
 		//register custom metrics with jmx
 		MetricscConfigImpl metricConfigMBean = new MetricscConfigImpl(noOfNewPatients, noOfEncounters);
 		MBeanServer mbs = ManagementFactory.getPlatformMBeanServer();
 		try {
-			ObjectName objectName = new ObjectName("com.sysdig.app:name=SystemStatusExample");
-			mbs.registerMBean(metricConfigMBean, objectName);
+			objectName = new ObjectName("com.sysdig.app:name=SystemStatusExample");
+			//			mbs.registerMBean(metricConfigMBean, objectName);
 		}
 		catch (MalformedObjectNameException e) {
 			LOGGER.error(e.getMessage());
 			throw new MetricsException(e);
 		}
-		catch (InstanceAlreadyExistsException e) {
-			LOGGER.error(e.getMessage());
-			throw new MetricsException(e);
-		}
-		catch (NotCompliantMBeanException e) {
-			LOGGER.error(e.getMessage());
-			throw new MetricsException(e);
-		}
-		catch (MBeanRegistrationException e) {
-			LOGGER.error(e.getMessage());
-			throw new MetricsException(e);
-		}
+		//		catch (InstanceAlreadyExistsException e) {
+		//			LOGGER.error(e.getMessage());
+		//			throw new MetricsException(e);
+		//		}
+		//		catch (NotCompliantMBeanException e) {
+		//			LOGGER.error(e.getMessage());
+		//			throw new MetricsException(e);
+		//		}
+		//		catch (MBeanRegistrationException e) {
+		//			LOGGER.error(e.getMessage());
+		//			throw new MetricsException(e);
+		//		}
```

Comment:
I'd tend to prefer just deleting this rather than commenting it out. Source control can keep track of what was there for you (have a look at the [Blame button on Github](https://github.com/openmrs/openmrs-module-metrics/blame/master/api/src/main/java/org/openmrs/module/metrics/MetricsActivator.java)... it's really cool!

Reply:
Oh yeah blame is always cool thanks alot